src/Orc.Controls.Example/ViewModels/HorizontalNavigationStepBarViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace Orc.Controls.Example.ViewModels
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Catel.MVVM;
+    using Orc.Controls.Controls.StepBar.Models;
```

Comment:
These namespaces immediately tell me something is wrong. Please check the other controls on how they deal with namespaces.